mods/ra/rules/defaults.yaml

```diff
@@ -511,6 +511,32 @@
 			TopLeft: -512, -512
 			BottomRight: 512, 512
 
+^Gate:
+	Inherits: ^BasicBuilding
+	Valued:
+		Cost: 250
+	Health:
+		HP: 350
+	Armor:
+		Type: Heavy
+	LineBuildNode:
+		Types: concrete, gate
+	-Building:
+	-WithSpriteBody:
+	WithGateSpriteBody:
+	Power:
```

Comment:
Why are you allowing people to power down gates, especially if they don't draw power?